ES/ESsdm021.xml

```diff
@@ -37,52 +37,166 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        <incipit xml:lang="gez">ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ ሊቃነ፡ መላእክት፡ ሚካኤል፡ ወገብርኤል፡ ሱራፌል፡ ወኪሩቤል፡
+                        </incipit>
+                        <note> cp. <bibl>
+                           <ptr target="bm:Aescoly1951Recueil"/>
+                           <citedRange unit="page">158-169</citedRange>
+                        </bibl>.</note>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i2.4">
+                        <locus from="42v" to="48r">42v-48r</locus>
+                        <title ref="LIT5927PrayerBE">Protective prayer</title>
+                        <incipit xml:lang="gez">ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ ወዝንቱ፡ ውእቱ፡ ስማየ፡ ዘነገርክዎ፡
+                        </incipit>
+                        <note> cp. <bibl>
+                           <ptr target="bm:Aescoly1951Recueil"/>
+                           <citedRange unit="page">176-185</citedRange>
+                        </bibl>.</note>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i2.5">
+                        <locus from="48r" to="49v">48r-49v</locus>
+                        <title>Incantation</title>
+                        <incipit xml:lang="gez">ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ ኤል፡ ኤል፡ ኤል፡ … ወእምብድብድ፡ ጽውዓ፡ ጽዋዕኩ፡
+                        </incipit>
+                        <note> cp. <bibl>
+                           <ptr target="bm:Aescoly1951Recueil"/>
+                           <citedRange unit="page">192-193</citedRange>
+                        </bibl>.</note>
+                     </msItem>
+                     <msItem xml:id="ms_i2.6">
+                        <locus from="49v" to="50v">49v-50v</locus>
+                        <title ref="LIT5929AsmatBE">ʾAsmāt-prayer</title>
+                        <incipit xml:lang="gez">ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ ወጸውኡ፡ ስማየ፡ በሳብዕት፡ ዕለት፡
+                        </incipit>
+                        <note> cp. <bibl>
+                           <ptr target="bm:Aescoly1951Recueil"/>
+                           <citedRange unit="page">186-187</citedRange>
+                        </bibl>.</note>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i2.7">
+                        <locus from="50v" to="55v">50v-55v</locus>
+                        <title ref="LIT5930PrayerBE">Prayer</title>
+                        <incipit xml:lang="gez">ወይእዜኒ፡ እግዚኦ፡ እግዚአብሔር፡ አሜን፡ ቡሩክ፡
+                        </incipit>
+                        <note> cp. <bibl>
+                           <ptr target="bm:Halevy1877Prieres"/>
+                           <citedRange unit="page">50-55</citedRange>
+                        </bibl>.</note>
+                        <colophon xml:id="coloph1"><locus target="#55v"></locus><note>Embedded colophon mentions <persName ref="PRS8321saggaAm">Ṣaggā ʾAmlāk</persName>, one of the Beta ʾƎsraʾel forefathers</note></colophon>
```

Comment:
If by "embedded colophon", colophon of the work is meant, this information would belong to the work record...I don't know in how much detail you wanted to process these records.